src/Components/Web/src/Forms/InputFile.cs

```diff
@@ -73,6 +73,11 @@ internal async ValueTask<IBrowserFile> ConvertToImageFileAsync(BrowserFile file,
         {
             var imageFile = await JSRuntime.InvokeAsync<BrowserFile>(InputFileInterop.ToImageFile, _inputFileElement, file.Id, format, maxWidth, maxHeight);
 
+            if (imageFile is null)
+            {
+                throw new InvalidOperationException("ToImageFile returned an unexpected null result.");
+            }
+
```

Comment:
I think this check is redundant. We control the code for `ToImageFile`, and we know it doesn't return null. Checking for null here is equivalent to checking nulls after arbitrary calls to our own private .NET methods that we know don't return null.

Reply:
Getting a NRE on the following line of code would be just as useful for detecting bugs or misuse.